server/src/main/resources/db/migration/V1_11__Denormalization.sql

```diff
@@ -0,0 +1,105 @@
+SET SCHEMA 'public';
+
+--
+-- DEPENDENCIES
+--
+
+ALTER TABLE extension_version
+    ADD dependencies character varying(1024);
+
+UPDATE extension_version v
+SET dependencies = tmp.dependencies
+FROM (
+    SELECT v.id, string_agg(v.fqn, ',') as dependencies FROM (
+        SELECT DISTINCT v.id, n.name || '.' || dep.name as fqn
```

Comment:
generally we can't get 100% sure that name & namespace contain no separator ','

Reply:
name and namespace are validated on publishing:
https://github.com/eclipse/openvsx/blob/1679a787dae9fee32ed3607d14a5af0acae0f336/server/src/main/java/org/eclipse/openvsx/ExtensionValidator.java#L50